pkg/service/id_kvstore.go

```diff
@@ -211,7 +211,7 @@ func acquireGlobalID(l3n4Addr loadbalancer.L3n4Addr, baseID uint32) (*loadbalanc
 	svcPath := path.Join(common.ServicesKeyPath, sha256Sum)
 
 	// Lock that sha256Sum
-	lockKey, err := kvstore.LockPath(context.Background(), svcPath)
+	lockKey, err := kvstore.LockPath(context.Background(), kvstore.Client(), svcPath)
```

Comment:
hmm, I wonder how we do this on master now. Still, looks like the correct conflict resolve